### PR TITLE
Add support for Terminal resources

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @ob-stripe 
cc @stripe/api-libraries @stevene-stripe

### CODE REVIEW

File: src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        public ConnectionTokenService(string apiKey)
+            : base(apiKey)
+        {
+        }
+
+        public override string BasePath => "/terminal/connection_tokens";
+
+        public virtual ConnectionToken Create(ConnectionTokenCreateOptions options, RequestOptions requestOptions = null)
```
Comment (ob-stripe) [inline src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs]: I wonder if we should provide overloads with no options. It seems a little bit silly to force users to create a `ConnectionTokenCreateOptions` instance that contains nothing.

Comment (remi-stripe) [reply src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs]: I hear you but then it becomes a mess of overloads and I'm not a big fan, we saw what it did to stripe-java.

**Comment by remi-stripe** [comment]
@ob-stripe Already did for the `StripeObjectConverter` part, likely while you reviewed

File: src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs (hunk 1 above)
Comment (ob-stripe) [reply src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs]: Just tested it, and we can simply set `null` as the default value for the `options` argument. It still satisfies the interface and makes the argument optional without requiring an overload.

Comment (remi-stripe) [reply src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs]: fixed, PTAL

**Comment by ob-stripe** [comment]
Released as 21.2.0.

**Pull request merged**